app/service/operation_svc.py

```diff
@@ -46,3 +53,4 @@ def __init__(self, data_svc, utility_svc, planner):
             link = dict(op_id=c['op_id'], host_id=c['agent_id'], ability_id=c['ability_id'], decide=datetime.now(),
                         command=c['command'], score=0, jitter=1)
```

Comment:
seems like lines 41-45 could call the self.close_operation, right?

Reply:
It actually did at one point in the past. Looks like a rebase undid that, though. Fixed.